e2e/create_test_component.go

```diff
@@ -18,256 +16,256 @@ import (
 )
 
 var (
-	basePath                     = "testdata/common"
+	basePath                     = "testdata/"
 	componentNamePrefix          = "component.name."
 	podinfoComponentName         = "/podinfo"
 	podinfoBackendComponentName  = "/podinfo/backend"
 	podinfoFrontendComponentName = "/podinfo/frontend"
 	redisComponentName           = "/redis"
+	podinfoName                  = "podinfo"
+	backend                      = "backend"
+	frontend                     = "frontend"
+	redis                        = "redis"
+	errMsgReadFile               = "failed to read file: %w"
 )
 
-func createTestComponentVersionUnsigned(t *testing.T, componentNameIdentifier string) *features.FeatureBuilder {
+func createTestComponentVersionUnsigned(t *testing.T, componentNameIdentifier string, testPath string, version string) *features.FeatureBuilder {
 	t.Helper()
 	return features.New("Add components to component-version").
-		Setup(setup.AddComponentVersions(podinfoBackend(t, componentNameIdentifier))).
-		Setup(setup.AddComponentVersions(podinfoFrontend(t, componentNameIdentifier))).
-		Setup(setup.AddComponentVersions(podinfoRedis(t, componentNameIdentifier))).
-		Setup(setup.AddComponentVersions(podinfo(t, componentNameIdentifier)))
+		Setup(setup.AddComponentVersions(podinfoBackend(t, nil, "", componentNameIdentifier, testPath, version))).
```

Comment:
I liked the basic singed component version better, as this just adds complexity to the calling function.

Reply:
We want the e2e tests to be as close to a demo as possible, so that colleagues in other teams can refer and compare when they start configuring.

Reply:
You can still do that with the basic signing component. The signing can be tested with a super simple component. The main path can be done with a complex component. But there is no need for that when you are testing verification.